python/src/main/python/drivers/glsl-to-spv-worker.py

```diff
@@ -120,22 +125,22 @@ def glsl2spv(glsl, spv):
 
 
 def prepare_shaders(frag_file, frag_spv_file, vert_spv_file):
-
     # Vert
     prepare_vert_file()
     glsl2spv('test.vert', vert_spv_file)
 
     # Frag
     glsl2spv(frag_file, frag_spv_file)
 
+
 ################################################################################
 
 
 def run_spirv_opt(spv_file, args):
     print("Running optimizer.")
     spv_file_opt = spv_file + '.opt'
     cmd = os.path.dirname(HERE) + '/../../bin/' + get_bin_type() + '/spirv-opt ' + \
-        args.spirvopt + ' ' + spv_file + ' -o ' + spv_file_opt
+          args.spirvopt + ' ' + spv_file + ' -o ' + spv_file_opt
```

Comment:
This now shows a PEP 8 (i.e. style) failure and there are others throughout this file. It would be good to fix them all and/or decide if we are out-of-sync somehow.

Reply:
OK we might be out of sync. At least, my line length is 100, which I think is inherited from the root project, but this might not apply if you are using PyCharm from within the `python` directory.

Reply:
I have fixed these